fe/fe-core/src/main/java/org/apache/doris/catalog/HdfsStorageVault.java

```diff
@@ -85,17 +90,66 @@ public HdfsStorageVault(String name, boolean ifNotExists, boolean setAsDefault)
     }
 
     @Override
-    public void modifyProperties(Map<String, String> properties) throws DdlException {
-        for (Map.Entry<String, String> kv : properties.entrySet()) {
+    public void modifyProperties(Map<String, String> newProperties) throws DdlException {
+        for (Map.Entry<String, String> kv : newProperties.entrySet()) {
             replaceIfEffectiveValue(this.properties, kv.getKey(), kv.getValue());
         }
+        checkConnectivity(this.properties);
     }
 
     @Override
     public Map<String, String> getCopiedProperties() {
         return Maps.newHashMap(properties);
     }
 
+    public static void checkConnectivity(Map<String, String> newProperties) throws DdlException {
+        if (newProperties.containsKey(S3Properties.VALIDITY_CHECK)
+                && newProperties.get(S3Properties.VALIDITY_CHECK).equalsIgnoreCase("false")) {
+            return;
+        }
+
+        String hadoopFsName = null;
+        String pathPrefix = null;
+        for (Map.Entry<String, String> property : newProperties.entrySet()) {
+            if (property.getKey().equalsIgnoreCase(HADOOP_FS_NAME)) {
+                hadoopFsName = property.getValue();
+            } else if (property.getKey().equalsIgnoreCase(VAULT_PATH_PREFIX)) {
+                pathPrefix = property.getValue();
+            }
+        }
+        Preconditions.checkArgument(!Strings.isNullOrEmpty(hadoopFsName), "%s is null or empty", HADOOP_FS_NAME);
+        Preconditions.checkArgument(!Strings.isNullOrEmpty(pathPrefix), "%s is null or empty", VAULT_PATH_PREFIX);
+
+        try (DFSFileSystem dfsFileSystem = new DFSFileSystem(newProperties)) {
+            String remotePath = hadoopFsName + "/" + pathPrefix + "/checkConnectivity";
+            Status st = dfsFileSystem.makeDir(remotePath);
```

Comment:
What is `checkConnectivity` already exists?

Reply:
> What is `checkConnectivity` already exists?

Add a timestamp for the check path to solve the problem